parsec/api/protocol/handshake.py

```diff
@@ -461,8 +476,16 @@ def __init__(
     def process_challenge_req(self, req: bytes) -> bytes:
         self.load_challenge_req(req)
         challenge = self.challenge_data["challenge"]
+
         assert isinstance(challenge, bytes)
-        answer = self.user_signkey.sign(challenge)
+
+        # TO-DO remove the else for the next release
+        if self.backend_api_version.version >= 2 and self.backend_api_version.revision >= 5:
```

Comment:
The following code looks a bit cleaner:
 `if self.backend_api_version >= (2, 5)`